src/lib/ngsiNotify/QueueNotifier.cpp

```diff
@@ -100,8 +101,8 @@ void QueueNotifier::sendNotifyContextRequest(NotifyContextRequest* ncr, const st
     spathList = "";
   }
 
-  ci.outFormat = format;
-  std::string payload = ncr->render(&ci, NotifyContext, "");
+  ci.outFormat = JSON;
```

Comment:
No, that is not the same type.
ci.outFormat can be either text/plain or application/json.
This is about the 'language' to use for the response, not about the internal formatting 'inside a language', like 'notifyFormat'.

Of course, we could join all of these variables to use the same type, somehow include 'text/plain' also and have ci.inFormat and ci.outFormat use that same type ...  I don't think it's a bad idea.
[ The three different notification 'formats' for json really complicated things ... :-D ]